spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+Rails::Controller::Testing.install
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get '/users'
+      expect(response).to be_successful
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      get '/users'
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] Great job adding Spec tests for Users controller. As per project requirements, correct view template is rendered for the `show` action and not `index` action as is your case with code snippet below
-   `expect(response).to render_template(:index)`

Kindly ensure the `show` action is tested with `describe 'GET /show' do`


![Screenshot from 2023-02-15 07-47-17](https://user-images.githubusercontent.com/62849872/218953901-ff9aad70-afb3-4a64-8d8b-822c88aa7c7e.png)